source/manual/review-page.html.md

```diff
@@ -26,20 +24,14 @@ We also document Icinga alerts. These explain what to do when a certain alert tr
 - Update the page
 - Remove it (don’t forget to [set up a redirect][redirects])
 - Assign it for someone else to review
```

Comment:
I don't think this makes sense any more, since there's no concept of "assignment". We could say: "Ask someone who has more context to review it".